static/templates/chat/action/header.hbs

```diff
@@ -0,0 +1,9 @@
+<h4 class="action">
```

Comment:
Is the `h4` styled such that a `strong` element will still be bolder?

Reply:
Yes - bolder is better! Actually I just kept to what we have now, since I didn't want to go hunt down CSS stuff as well.